.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
   - "7"
+before_install:
+  - sudo apt-get -qq install -y luarocks
+  - sudo luarocks install luacheck
+script:
+  - luacheck --codes --ranges -q game/scripts/vscripts; exit $(($? - 1))
```

Comment:
I think you need to adds in `npm run test` here since script override the default script to run (language node sets up npm run test automatically)

Looking at the travis output, the node test cases never ran